android/java/org/chromium/chrome/browser/preferences/website/BraveShieldsContentSettings.java

```diff
@@ -91,6 +92,9 @@ public static void setShieldsValue(Profile profile, String host, String resource
         if (resourceIndentifier.equals(RESOURCE_IDENTIFIER_FINGERPRINTING)) {
             BraveShieldsContentSettingsJni.get().setFingerprintingControlType(
                     settingOption, host, profile);
+        } else if (resourceIndentifier.equals(RESOURCE_IDENTIFIER_HTTPS_UPGRADE)) {
+            BraveShieldsContentSettingsJni.get().setHttpsUpgradeControlType(
+                    settingOption, host, profile);
```

Comment:
Is https://github.com/brave/brave-core/blob/a8fa5589278dbf11033260aea82c72e12bed26cc/android/java/org/chromium/chrome/browser/preferences/website/BraveShieldsContentSettings.java#L84 still needed if we have this HTTPS upgrade setting now?

Reply:
We will be removing HTTPS Everywhere support in the future, once this is fully deployed.